po/common/common.pot

```diff
@@ -5279,8 +5279,8 @@ msgid "Abbreviated product name"
 msgstr "Abbreviated product name"
 
 msgctxt "abbreviated_product_name_note"
-msgid "Product name with abbreviations shown on receipts"
-msgstr "Product name with abbreviations shown on receipts"
+msgid "Abbreviated product name as shown on receipts"
+msgstr "If a consumer buys a product in a shop, he will get a payment ticket. The product name on the ticket will be abbreviated."
```

Comment:
msgstr should be empty, is this a comment?

Reply:
It was not empty, the one above as well? I'll add it as a comment (not useful for Crowdin).